Accessibility: add Overview topic (#5042)
DevExpress/devextreme-documentation: concepts/Common/Accessibility/05 Documentation for Components.md

> The following list can help you find detailed information about each component's accessibility compliance.

Reviewer: ```suggestion
Refer to the following list for detailed information about each component's accessibility compliance:
```

> [note] At the moment, not all components' guides include an accessibility topic. This list will continue to grow.

Reviewer: ```suggestion
[note] Currently, not all components' guides include an accessibility topic. This list will continue to grow.
```
